src/org/labkey/cds/CDSModule.java

```diff
@@ -70,6 +85,10 @@
     final ModuleProperty _mabReport2ID;
     final ModuleProperty _mabReport2Label;
 
+    //wiki properties for the front page
+    final ModuleProperty _whatYouNeedToKnowWiki;
+    final ModuleProperty _takeATourWiki;
+
```

Comment:
@labkey-bpatel 
Do these need to be member variables? They don't appear to be used elsewhere in this class.

Reply:
They don't need to be member variables, however, there are other ModuleProperty variables above it declared similarly so thought to follow the same pattern.